templates/global/components/user_box.html

```diff
@@ -0,0 +1,48 @@
+{% load static %}
+{% load socialaccount %}
+
+
+<div class="p-2 border border-dark rounded bg-white">
+    {% if user.is_authenticated %}
+
+        <div class="d-flex align-items-center text-truncate">
+            <span class="me-2">
+                <!-- <a href="https://iconscout.com/icons/user" target="_blank">User Icon</a> 
+                    by <a href="https://iconscout.com/contributors/gajah-mada-studio">Mada Creative</a> 
+                    on <a href="https://iconscout.com">IconScout</a> -->
+                <img src="{{ user.socialaccount_set.all.0.get_avatar_url|default:"/static/icon/user.svg" }}" 
+                    alt="Icona Google di {{ user.username }}" 
+                    class="rounded-circle border border-secondary border-2" 
+                    style="height: 50px; width: 50px; background-color:white;" />
+            </span>
+                
+            <span>
+                <h5 class="mb-0">{{ user.socialaccount_set.all.0.extra_data.name|default:user.username }}</h5>
+                <p class="mb-0 text-muted " style="font-size:small;">{{ user.socialaccount_set.all.0.extra_data.email }}</p>
+            </span>
+
+        </div>
+
+
```

Comment:
Bene, nota grafica: è necessario allinearlo; da decidere se allinearlo con il nome oppure in fondo a destra